scripts/fix_files.sh

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Oh yes thanks you are right, I've been used to force the shell path for 25y :) Like a muscle memory. I'll update all my scripts for use env instead!